pkg/cloud/services/networking/port.go

```diff
@@ -58,7 +58,7 @@ func (s *Service) GetPortFromInstanceIP(instanceID string, ip string) ([]ports.P
 	return s.client.ListPort(portOpts)
 }
 
-func (s *Service) CreatePort(eventObject runtime.Object, clusterName string, portName string, portOpts *infrav1.PortOpts, instanceSecurityGroups []string, instanceTags []string) (*ports.Port, error) {
+func (s *Service) CreatePort(eventObject runtime.Object, clusterName string, portName string, portOpts *infrav1.PortOpts, defaultSecurityGroups []string, baseTags []string) (*ports.Port, error) {
```

Comment:
Just a minor notice about the name `defaultSecurityGroups`. Since there is a default security group inside Openstack this may cause confusion at first glance.

Maybe there is a good reason for not sticking with the `managedSecurityGroups` name from the openstackmachine_controller?

Reply:
what about naming it `securityGroups`. We now create ports with SGs, so to me the function arg should be named like this.

Reply:
good idea :+1:

Reply:
It's internal API anyway, I'm fine.

Reply:
Agree, we can levar it as is after the explanation